app/startup.go

```diff
@@ -70,7 +70,10 @@ func (app *App) startup(ctx context.Context) error {
 
 	app.initStartup(ctx, "Startup.Slack", app.initSlack)
 	app.notificationManager.RegisterSender(notification.DestTypeUserEmail, "smtp", email.NewSender(ctx))
-	app.notificationManager.RegisterSender(notification.DestTypeUserWebhook, "webhook", webhook.NewSender(ctx))
+	app.notificationManager.RegisterSender(notification.DestTypeUserWebhook, "userWebhook", webhook.NewSender(ctx))
+	if expflag.ContextHas(ctx, expflag.ChanWebhook) {
+		app.notificationManager.RegisterSender(notification.DestTypeChanWebhook, "chanWebhook", webhook.NewSender(ctx))
```

Comment:
Since these strings are primarily used for logging, let's use a hyphen for scan-ability and to match the style for Twilio SMS vs voice of `type-subtype`

```suggestion
	app.notificationManager.RegisterSender(notification.DestTypeUserWebhook, "webhook-user", webhook.NewSender(ctx))
	if expflag.ContextHas(ctx, expflag.ChanWebhook) {
		app.notificationManager.RegisterSender(notification.DestTypeChanWebhook, "webhook-channel", webhook.NewSender(ctx))
```